CMakeLists.txt

```diff
@@ -103,13 +118,13 @@ set(CPACK_RPM_PACKAGE_AUTOREQPROV ON)
 
 include(CPack)
 
-INSTALL(TARGETS hiredis hiredis_static
+INSTALL(TARGETS ${HIREDIS_TARGETS}
     EXPORT hiredis-targets
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-if (MSVC)
+if (MSVC AND BUILD_SHARED_LIBS)
```

Comment:
I missed two others;
- a similar change for `hiredis_ssl` on line 228
- a non-checked use of `hiredis_ssl` on line 217 (which in legacy is a bit weird, but maybe needed)

Reply:
Thanks, I updated the PR